pkg/kubernetes/utils.go

```diff
@@ -3,14 +3,95 @@ package kubernetes
 import (
 	"fmt"
 	"io/ioutil"
+	"log"
+	"os"
+	"path"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	networkv1 "k8s.io/api/networking/v1"
+	jsonserializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
+
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+var mDir string = "../../deployments"
```

Comment:
If we set the env variable for mDir, we don't have to use global variable! 🙇🏼‍♂️

Reply:
thank you! i fixed it!